etl/steps/data/grapher/who/2024-07-26/mortality_database.py

```diff
@@ -16,7 +14,7 @@ def run(dest_dir: str) -> None:
     # Load inputs.
     #
     # Load garden dataset.
-    ds_garden = cast(Dataset, paths.load_dependency("mortality_database"))
+    ds_garden = paths.load_dataset("mortality_database")
 
     # Read table names from garden dataset.
     table_names = sorted(ds_garden.table_names)
```

Comment:
Maybe there is a good reason not to do it, but I would pull all this in a list comprehension like:
tables = [ds_garden[tab] for tab in sorted(ds_garden.table_names)]

and then ds_grapher = create_dataset(dest_dir, tb=tables ...)

Reply:
Ah very good point, shall do this